.circleci/config.yml

```diff
@@ -93,3 +122,13 @@ workflows:
     jobs:
       - sfdx-docker-slim
       - sfdx-docker-full
+  nightly-release-workflow:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - validate-for-release
```

Comment:
Every night at midnight we will check to see if there is a different stable version of the CLI then what is published in Docker, if so publish.